x-pack/plugins/case/server/client/cases/update.ts

```diff
@@ -334,175 +337,189 @@ export const update = async ({
+      const currentCase = myCases.saved_objects.find((c) => c.id === caseToUpdate.id);
+      return (
+        currentCase != null &&
+        caseToUpdate.settings?.syncAlerts != null &&
+        currentCase.attributes.settings.syncAlerts !== caseToUpdate.settings.syncAlerts &&
+        caseToUpdate.settings.syncAlerts
+      );
+    });
 
-  // Update the alert's status to match any case status or sync settings changes
-  await updateAlerts({
-    casesWithStatusChangedAndSynced,
-    casesWithSyncSettingChangedToOn,
-    caseService,
-    client: savedObjectsClient,
-    caseClient,
-    casesMap,
-  });
+    // Update the alert's status to match any case status or sync settings changes
+    await updateAlerts({
+      casesWithStatusChangedAndSynced,
+      casesWithSyncSettingChangedToOn,
+      caseService,
+      client: savedObjectsClient,
+      caseClient,
+      casesMap,
+    });
 
-  const returnUpdatedCase = myCases.saved_objects
-    .filter((myCase) =>
-      updatedCases.saved_objects.some((updatedCase) => updatedCase.id === myCase.id)
-    )
-    .map((myCase) => {
-      const updatedCase = updatedCases.saved_objects.find((c) => c.id === myCase.id);
-      return flattenCaseSavedObject({
-        savedObject: {
-          ...myCase,
-          ...updatedCase,
-          attributes: { ...myCase.attributes, ...updatedCase?.attributes },
-          references: myCase.references,
-          version: updatedCase?.version ?? myCase.version,
-        },
+    const returnUpdatedCase = myCases.saved_objects
+      .filter((myCase) =>
+        updatedCases.saved_objects.some((updatedCase) => updatedCase.id === myCase.id)
+      )
+      .map((myCase) => {
+        const updatedCase = updatedCases.saved_objects.find((c) => c.id === myCase.id);
+        return flattenCaseSavedObject({
+          savedObject: {
+            ...myCase,
+            ...updatedCase,
+            attributes: { ...myCase.attributes, ...updatedCase?.attributes },
+            references: myCase.references,
+            version: updatedCase?.version ?? myCase.version,
+          },
+        });
       });
-    });
 
-  await userActionService.postUserActions({
-    client: savedObjectsClient,
-    actions: buildCaseUserActions({
-      originalCases: myCases.saved_objects,
-      updatedCases: updatedCases.saved_objects,
-      actionDate: updatedDt,
-      actionBy: { email, full_name, username },
-    }),
-  });
+    await userActionService.postUserActions({
+      client: savedObjectsClient,
+      actions: buildCaseUserActions({
+        originalCases: myCases.saved_objects,
+        updatedCases: updatedCases.saved_objects,
+        actionDate: updatedDt,
+        actionBy: { email, full_name, username },
+      }),
+    });
 
-  return CasesResponseRt.encode(returnUpdatedCase);
+    return CasesResponseRt.encode(returnUpdatedCase);
+  } catch (error) {
+    const idVersions = cases.cases.map((caseInfo) => ({
```

Comment:
`cases.cases` lol. Thoughts on updating this naming pattern in the future?

Reply:
haha 💯  I'll add it to the refactoring list.